code/game/objects/structures/stool_bed_chair_nest/stools.dm

```diff
@@ -11,11 +11,13 @@
 			qdel(src)
 			return
 		if(2.0)
-			if (prob(50))
+			if (prob(70))
+				new /obj/item/stack/sheet/metal(loc)
 				qdel(src)
 				return
 		if(3.0)
-			if (prob(5))
+			if (prob(50))

```

Comment:
Ergo-Work Are there any other code comments on #11382 because every time someone adds one I die a little on the inside